deploy/sample-backup-configs/sample_config_hwdb.yaml

```diff
@@ -10,26 +10,26 @@
 # own images.
 
 firesim_supernode_rocket_singlecore_nic_l2_lbp:
-    agfi: agfi-0d4da869862e1cd50
+    agfi: agfi-02d92ba19f0ccc4d8
     deploy_triplet_override: null
     custom_runtime_config: null
+
 firesim_boom_singlecore_nic_l2_llc4mb_ddr3:
-    agfi: agfi-06dab9d0f685825af
+    agfi: agfi-09628426dfa714976
     deploy_triplet_override: null
     custom_runtime_config: null
+
 firesim_boom_singlecore_no_nic_l2_llc4mb_ddr3:
-    agfi: agfi-072e726636c7fea48
+    agfi: agfi-0dfa5d15a4becb5db
     deploy_triplet_override: null
     custom_runtime_config: null
+
 firesim_rocket_quadcore_nic_l2_llc4mb_ddr3:
-    agfi: agfi-06e0e0eab95b05fde
+    agfi: agfi-081255e6e4e2d3fcb
     deploy_triplet_override: null
     custom_runtime_config: null
+
 firesim_rocket_quadcore_no_nic_l2_llc4mb_ddr3:
-    agfi: agfi-0090299c30c31b8f7
-    deploy_triplet_override: null
-    custom_runtime_config: null
-firesim_supernode_rocket_singlecore_nic_l2_lbp:
-    agfi: agfi-07f2cc8506789232c
+    agfi: agfi-080285efd8520fb27
```

Comment:
Does this mean that you regenerated AGFI's with this PR or was the merge somehow messed up?  Just curious.

Reply:
I regenerated AGFIs with the most recent commits just to verify that everything was working. I then noticed that there was a duplicate HWDB entry so I figured I would just update everything.